funcx_endpoint/tests/unit/test_register_endpoint.py

```diff
@@ -99,3 +99,45 @@ def test_register_endpoint_invalid_response(
             endpoint_dir=tmp_path,
             endpoint_name="Some EP Name",
         )
+
+
+@responses.activate
+@pytest.mark.parametrize("multi_tenant", [None, True, False])
+def test_register_endpoint_multi_tenant(
+    tmp_path,
+    endpoint_uuid,
+    other_endpoint_id,
+    register_endpoint_response,
+    get_standard_funcx_client,
+    randomstring,
+    multi_tenant,
+):
+    ep_uuid = str(uuid.uuid4())
+    uname, pword = randomstring(), randomstring()
+    register_endpoint_response(endpoint_id=ep_uuid, username=uname, password=pword)
```

Comment:
FWIW, it doesn't look like this test uses any of the arguments, so could use the defaults:
```diff
- ep_uuid = str(uuid.uuid4())
- uname, pword = randomstring(), randomstring()
- register_endpoint_response(endpoint_id=ep_uuid, username=uname, password=pword)
+ register_endpoint_response(endpoint_id=endpoint_uuid)
```